### PR TITLE
Add information about Verification of Findings initiative

### PR DESCRIPTION
**Pull request by fcanogab**
- Information about the Verification of Findings initiative.
- First version of guidelines to be reviewed before reporting a vulnerability to an open source project.

### CODE REVIEW

**Comment by coderpatros** [comment]
Maybe a recommendation that creating a public github issue is perhaps not the best way to report a vulnerability? And that you should check for a security policy or another channel to responsibly disclose?

File: publications/verification-of-findings/guidelines.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+* Explicitly state which versions do you think are not affected and why.
+* Include a brief description of the vulnerability.
+* Explain where did you get the software and how you installed and configured it.
+* Include step by step instructions about how to reproduce the vulnerability.
+* If you are capable of doing it, include a reproducer. A reproducer is a piece of code that demonstrates the vulnerability.
+* As we are talking about open source, indicate which lines of the code are responsible for the vulnerability.
+* In the same sense than the reproducer, if you are able, include a potential patch for the vulnerability.
+* Include information about how to contact you (e.g. email).
+* Be ready to answer the maintainer's questions.
+* State the vulnerability in a way that the risk for the user or organizatoin is clear. Avoid justifying that it is a security vulnerability because it is a security vulnerability (an SQL injection is not dangerous because it is a security vulnerability; it is dangerous because it allows an unauthorized user to see and change confidential data).
```
Comment (psiinon) [inline publications/verification-of-findings/guidelines.md:19]: 'organizatoin' typo.

File: publications/verification-of-findings/guidelines.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    * If it does not represent a risk or affects security pillars (confidentiality, integrity, availability, authenticity, accountability, non-repudiation or reliability), it might not be a security issue, but just a functionality issue.
+* Include screenshots, crash reports, demo videos, or any other helpful material.
+    * A crash report or a video is not a substitute for a good explanation, a reproducer or even a patch. Never send only a crash report or video.
```
Comment (psiinon) [inline publications/verification-of-findings/guidelines.md:22]: For the bug bounty programs I've been involved with (Mozilla and ZAP) videos are strongly discouraged. Its almost impossible to get any useful info out of them.

File: publications/verification-of-findings/guidelines.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* Be sure that the vulnerability you see is not a problem in your own configuration.
+* If you know there is a vulnerability because the version of the package, be careful.
+    * Just looking at the version number of a package will not tell us if they are vulnerable or not. For example, PHP 5.3 may be vulnerable or not to CVE-2014-3670, depending on where you have downloaded it from due to backporting (https://access.redhat.com/security/updates/backporting).
+    * You can verify this by trying to exploit the vulnerability. If you cannot exploit it, please, think twice before reporting it.
+* Use search engines in order to try to identify if other people have already identified what you have found, and it is a known functionality or issue.
+* If you found the vulnerability by executing a tool, try to verify manually the existence of the vulnerability. Security tools give false positives sometimes.
```
Comment (psiinon) [inline publications/verification-of-findings/guidelines.md:28]: I'd put it stronger thatn that - always try to manually verify and tool findings.

File: publications/verification-of-findings/guidelines.md (hunk 1 above)
Comment (psiinon) [reply publications/verification-of-findings/guidelines.md:19]: And in the line below;)

**Comment by david-a-wheeler** [comment]
This looks like it overlaps with the OpenSSF Vulnerability Disclosures WG: <https://github.com/ossf/wg-vulnerability-disclosures> - I recommend coordinating with them.

**Comment by david-a-wheeler** [comment]
Also: If it's an OSS project, in many cases OSS projects expect source code changes.

**Comment by david-a-wheeler** [comment]
At the least, reproducibility is a minimum bar; without that, it's impossible to determine if it's there or if it's fixed.

**Comment by ware** [comment]
I'd like to discuss where we are with this discussion at the WG meeting this week.

**Comment by fcanogab** [comment]
@ware we talked that it was probably better to include this in the Vulnerability Disclosure WG and in that group they thought this has not got enough significance to be a project by itself. I think we can close this issue.

**Pull request closed without merging**